trading_calendars/exchange_calendar_xnys.py

```diff
@@ -47,9 +47,9 @@
 # http://www.stevemorse.org/jcal/whendid.html
 
 
-class NYSEExchangeCalendar(TradingCalendar):
+class XNYSExchangeCalendar(TradingCalendar):
     """
-    Exchange calendar for NYSE
+    Exchange calendar for NYSE (XNYS).
```

Comment:
Should we just write this out as "New York Stock Exchange" to match the other calendars?